modules/counters.js

```diff
@@ -1,3 +1,23 @@
+var QUERY_TOTAL_USERS = "select total from overall.total";
+
+var request = require('superagent');
+
 module.exports = function(app, router){
 
+  router.get('/ctl', function(req, res) {
+    request
+     .get("http://crisistextline.org/trends/data/messages.txt")
+     .end(function(response){
+        var total = response.text;
+        res.json({total: total});
+     });
+  });
+
+  router.get('/dosomething', function(req, res) {
+    app.service_data.performQuery(QUERY_TOTAL_USERS, function(rows) {
```

Comment:
I think this can be taken from the [Drupal API](https://github.com/DoSomething/dosomething/wiki/API#get-member-count), which might be more robust than directly querying SQL in this app.